Dockerfile

```diff
@@ -4,18 +4,19 @@ LABEL maintainer="team@semantic.works"
 
 RUN apt-get update && apt-get -y upgrade && apt-get -y install git openssh-client rsync
 
-ENV MU_SPARQL_ENDPOINT 'http://database:8890/sparql'
-ENV MU_APPLICATION_GRAPH 'http://mu.semte.ch/application'
-ENV NODE_ENV 'production'
+ENV MU_SPARQL_ENDPOINT='http://database:8890/sparql'
+ENV MU_APPLICATION_GRAPH='http://mu.semte.ch/application'
+ENV NODE_ENV='production'
 
-ENV HOST '0.0.0.0'
-ENV PORT '80'
+ENV HOST='0.0.0.0'
+ENV PORT='80'
 
-ENV LOG_SPARQL_ALL 'true'
-ENV DEBUG_AUTH_HEADERS 'true'
+ENV LOG_SPARQL_ALL='true'
+ENV DEBUG_AUTH_HEADERS='true'
 
 WORKDIR /usr/src/app
 COPY package.json /usr/src/app/package.json
+COPY package-lock.json /usr/src/app/package-lock.json
```

Comment:
use the package-lock file

Reply:
I doubt we can blindly drop this in and expect things to work correctly.  I vaguely recall the package-lock gave issues with the embedding of 'mu' and/or with people creating a package-lock file with a newer npm version than the one in the template which also created its own issues.  That's probably a different topic?

Reply:
see later, you may have run `npm ci` if you're seeing that behaviour?